src/main/resources/assessment-config/accommodation/accommodation-assessment-v2.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
```

Comment:
Might be worth making this more extensive (maybe like the prototype) so the demo is more interesting